stonesoup/types/state.py

```diff
@@ -20,12 +20,12 @@ class State(Type):
     Most simple state type, which only has time and a state vector."""
     timestamp: datetime.datetime = Property(
         default=None, doc="Timestamp of the state. Default None.")
-    state_vector: StateVector = Property(doc='State vector.')
+    state_vector: Union[StateVector, StateVectors] = Property(doc='State vector.')
 
     def __init__(self, state_vector, *args, **kwargs):
         # Don't cast away subtype of state_vector if not necessary
         if state_vector is not None \
-                and not isinstance(state_vector, StateVector):
+                and not isinstance(state_vector, (StateVector, StateVectors)):
```

Comment:
Noting comment above, for ease, probably best keep the addition of `StateVectors` here.